dask_bigquery/core.py

```diff
@@ -174,3 +200,111 @@ def make_create_read_session_request(row_filter="", max_stream_count=0):
 
         graph = HighLevelGraph({output_name: layer}, {output_name: set()})
         return new_dd_object(graph, output_name, meta, divisions)
+
+
+def to_gbq(
+    df,
+    *,
+    project_id: str,
+    dataset_id: str,
+    table_id: str,
+    bucket: str = "dask-bigquery-tmp",
+    credentials: Credentials = None,
+    delete_bucket: bool = False,
+    parquet_kwargs: dict = None,
+    load_job_kwargs: dict = None,
+):
+    """Write dask dataframe as table using BigQuery LoadJob.
+    Writes Parquet to GCS for intermediary storage.
+
+    Parameters
+    ----------
+    project_id: str
+      Name of the BigQuery project id.
+    dataset_id: str
+      BigQuery dataset within project
+    table_id: str
+      BigQuery table within dataset
+    bucket: str
+      Google Cloud Storage bucket name, for intermediary parquet storage. If the bucket doesn't
+      exist, it will be created. The account you're using will need Google Cloud Storage
+      permissions (storage.objects.create, storage.buckets.create). If a persistent bucket is used,
+      it is recommended to configure a retention policy that ensures
+      the data is cleaned up in case of job failures.
+      Default: dask-bigquery-tmp.
+    credentials : google.auth.credentials.Credentials, optional
+      Credentials for accessing Google APIs. Use this parameter to override
+      default credentials.
+    delete_bucket:
+      Delete bucket in GCS after loading intermediary data to Big Query.
+      Default: False.
+    parquet_kwargs: dict
+      Additional kwargs to pass to dataframe.write_parquet, such as schema, partition_on or
+      write_index. For writing parquet, pyarrow is required.
+      Default: {"write_index": False}
+    load_job_kwargs: dict
+      Additional kwargs to pass when creating bigquery.LoadJobConfig, such as schema,
+      time_partitioning, clustering_fields, etc.
+      Default: {"autodetect": True}
+
+    Returns
+    -------
+        LoadJobResult
+    """
+    if project_id is None and hasattr(credentials, "project_id"):
+        # service account credentials have a project associated with them
+        project_id = credentials.project_id
+    if not project_id:
+        raise ValueError("Required: project_id")
+    if not table_id:
+        raise ValueError("Required: table_id")
+    if not dataset_id:
+        raise ValueError("Required: dataset_id")
+
+    parquet_kwargs = parquet_kwargs or {}
+    if not parquet_kwargs:
+        parquet_kwargs["write_index"] = False
+
+    load_job_kwargs = load_job_kwargs or {}
+    if not load_job_kwargs:
+        load_job_kwargs["autodetect"] = True
+
+    fs = gcs_fs(project_id, credentials=credentials)
+    if fs.exists(bucket):
+        # if we didn't create it, we shouldn't delete it
+        delete_bucket = False
+    else:
+        fs.mkdir(bucket)
+
+    token = tokenize(df)
+    object_prefix = f"{dt.datetime.utcnow():%Y-%m-%dT%H-%M-%S}_{token}"
```

Comment:
I suggest to use isoformat instead of the custom string. it's more precise and standardized.

```suggestion
    object_prefix = f"{dt.datetime.utcnow().isoformat()}_{token}"
```